Core/Core/Discussions/DiscussionDetailsViewController.swift

```diff
@@ -448,7 +485,7 @@ extension DiscussionDetailsViewController {
             text-align: center;
             text-decoration: none;
         ">
-            \(CoreWebView.htmlString(NSLocalizedString("View more replies", bundle: .core, comment: "")))
+            \(t(NSLocalizedString("View more replies", bundle: .core, comment: "")))
```

Comment:
What does this `t` do?

Reply:
Oh...it's a method. I was scanning from the bottom up 😂

Reply:
yeah I was trying to trim up the really long common escaping.